packages/core/src/components/axes/toolbar.ts

```diff
@@ -401,6 +401,12 @@ export class Toolbar extends Component {
 
 	// Calls passed function && dispatches event
 	triggerFunctionAndEvent(control, event, element?) {
+		// Check if trigger is disabled
+		if (typeof control.shouldBeDisabled === 'function') {
+			if (control.shouldBeDisabled()) {
```

Comment:
why not have both conditions in 1 statement?

Reply:
Short circuiting, wasn't used in other scenarios, but will switch to 1 statement.